packages/components/src/styles/components/form/text-input.scss

```diff
@@ -128,7 +128,9 @@
   // we override the default icon with the Flight corresponding one
   // notice: the original in Chrome has two assets, one for light and one for dark mode, and uses a special syntax, but apparently it doesn't work if used in a stylesheet
   &[type="date"],
-  &[type="datetime-local"] {
+  &[type="datetime-local"],
```

Comment:
Does this change require a changelog?

Reply:
since we haven't released yet, there would be no actual difference, so no.